internal/kafka/internal/api/public/api/openapi.yaml

```
@@ -1208,6 +1209,7 @@ components:
         owner: api_kafka_service
         name: serviceapi
         bootstrap_server_host: serviceapi-1isy6rq3jki8q0otmjqfd3ocfrg.apps.mk-bttg0jn170hp.x5u8.s1.devshift.org
+        admin_api_server_url: admin-server-mk-e-e-e-e-c---{}ld{-}-n-vp--bltg.bf2.kafka-stage.rhcloud.com
```

Unsure about this line here @machi1990, specifically the url I have attatched.

---

You could add a protocol. 
```suggestion
        admin_api_server_url: https://admin-server-mk-e-e-e-e-c---{}ld{-}-n-vp--bltg.bf2.kafka-stage.rhcloud.com
```